spec/fizzbuzz_spec.rb

```diff
@@ -1,22 +1,31 @@
 require 'rspec'
 require 'fizzbuzz'
 
+def expect_specific(value, script)
+  lambda { |test_case| expect(script.run(test_case)).to eq value }
+end
+
 describe 'SuperFizzBuzz' do
   let(:script) { SuperFizzBuzz.new }
 
   it "returns 'Fizz' when my input is divisible by 3" do
-    expect(script.run(3)).to eq "Fizz"
+    expect_fizz = expect_specific("Fizz", script)
+
+    [3, -132, 213].each &expect_fizz
```

Comment:
This is a super-neat approach.  To achieve a similar effect, we'll often use a loop to generate different *contexts*.  For instance:

```ruby
[
  3,
  -132,
  213,
].each do |number|
  context 'when number is #{number} do
    it "returns 'Fizz'" do
      expect(SuperFizzBuzz.new.run(number)).to eq 'Fizz'
   end
  end 
```

or something like that